core/controlplane/config/templates/cloud-config-controller

```diff
@@ -47,17 +38,7 @@ coreos:
         Type=oneshot
         RemainAfterExit=true
         ExecStartPre=/bin/touch /etc/aws-environment
```

Comment:
Instead of having an `/opt/bin/get-stack-name`, you can have
```
Environment={{.StackNameEnvFileName}}
```

in unit files and then use `KUBE_AWS_STACK_NAME` and var inside scripts. adding `set -ue` to these scripts will make them fail loudly if vars are not set/not passed